highcharts-api/highcharts-maps/03-dynamic-drilldown/README.md

```diff
@@ -0,0 +1,28 @@
+# Maps Exercise 3 - Dynamic Drilldown
+​
+![Sample Map Drilldown Visualization](map-drilldown.gif)
+​
+Your task is to create a script that generates a dynamic map drilldown representation, akin to the one presented in the sample. The specific requirements are:
+​
+## 1. Base World Continent View
```

Comment:
I think the first requirement can be a little bit misleading because the basic demo already covers fetching topology. How about changing this to:

Based on the topology data, generate bogus values for World Continents series.

Reply:
I've applied the requested changes, but if you find anything else that can be changed/improved, feel free to commit to this branch.

Additionally, I wonder if we should add documentation links to `readme.md`. What do you think about it?

Reply:
Thank you! I was also curious about that too, because we follow a convention without including any API links. However, this is a bit tricky, so I think we can leave it as it is.